.github/workflows/test.yml

```
@@ -63,3 +63,55 @@ jobs:
 
       - name: Run unit tests
         run: composer phpunit
+
+  coverage:
+    needs: test
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php: [ '7.4' ]
+        experimental: [ false ]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          ini-values: zend.assertions=1, error_reporting=-1, display_errors=On
+          coverage: xdebug
+
+      - name: Install Composer dependencies - normal
+        uses: "ramsey/composer-install@v2"
+
+      - name: Run the unit tests with code coverage
+        run: vendor/bin/phpunit
```

```suggestion
        run: composer coverage
```

By using the Composer script, if this command would get additional arguments at some point, those only need to be maintained in one place.